integration-tests/virtual-threads/grpc-virtual-threads/pom.xml

```diff
@@ -93,6 +93,42 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>net.revelc.code</groupId>
+                <artifactId>impsort-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>sort-imports</id>
+                        <goals>
+                            <goal>sort</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <skip>${impsort.skip}</skip>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${version.surefire.plugin}</version>
+                <configuration>
+                    <systemPropertyVariables>
+                        <junit.quarkus.orderer.secondary-orderer>
+                            org.junit.jupiter.api.ClassOrderer$OrderAnnotation
+                        </junit.quarkus.orderer.secondary-orderer>
+                    </systemPropertyVariables>
```

Comment:
Is this really needed?

Reply:
I don't believe it's necessary for these tests.

Reply:
Yeah, I don't see it being of use iin the tests

Reply:
I've cleaned up the pom files.